app/views/admin/sites/_sites_neighbourhood_fields.erb

```diff
@@ -0,0 +1,2 @@
+<%= f.input :name %>
+<%= f.hidden_field :relation_type, value: "Secondary" %>
```

Comment:
lol ignore this file. didn't realise i hadn't deleted it. it was a misnamed file